src/app/pages/products/products.module.ts

```diff
@@ -0,0 +1,10 @@
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {ProductsComponent} from './products.component';
+import {RouterModule} from '@angular/router';
+
+@NgModule({
+  declarations: [ProductsComponent],
+  imports: [CommonModule, RouterModule.forChild([{path: '', component: ProductsComponent}])]
```

Comment:
Better to create a simple `route` constant above @NgModule and specify routes there (same structure as app-routing.module)

Reply:
Can you please give more details? I didn't get this one)
Do you mean in this file?

Reply:
Please look) Did I understand you well?

import {NgModule} from '@angular/core';
import {CommonModule} from '@angular/common';
import {ProductsComponent} from './products.component';
import {RouterModule, Routes} from '@angular/router';

const routes: Routes = [{path: '', component: ProductsComponent}];

@NgModule({
  declarations: [ProductsComponent],
  imports: [CommonModule, RouterModule.forChild(routes)]
})
export class ProductsModule {}

Reply:
yeah, so instead of directly specifying the routes:
```suggestion

const routes: Routes = [
  { path: '', component: ProductsComponent }
];

@NgModule({
  declarations: [ProductsComponent],
  imports: [CommonModule, RouterModule.forChild(routes)]
```

Reply:
Okay, already did that)